packages/cf.rb

```diff
@@ -3,7 +3,7 @@
 class Cf < Package
   description 'The official command line client for Cloud Foundry'
   homepage 'https://docs.cloudfoundry.org/cf-cli'
-  version '6.36.1'
+  version '6.36.1-1'
```

Comment:
New versions?
https://packages.cloudfoundry.org/stable?release=linux64-binary&version=7.2.0&source=github-rel
https://packages.cloudfoundry.org/stable?release=linux32-binary&version=7.2.0&source=github-rel

Reply:
@uberhacker would you mind doing this one? I can't figure it out.

Reply:
@saltedcoffii: I updated cf to the latest version.